File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidVerificationCodeError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class InvalidVerificationCodeError extends UserFacingError {

    public InvalidVerificationCodeError() {
        super(UserFacingErrorMessages.INSTANCE.invalidVerificationCode());
    }
}

File: android/apollo/src/main/java/io/muun/apollo/domain/errors/p2p/InvalidVerificationCodeError.kt
package io.muun.apollo.domain.errors.p2p


import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.UserFacingError

class InvalidVerificationCodeError : UserFacingError(
    UserFacingErrorMessages.INSTANCE.invalidVerificationCode()
)
